src/test/java/org/broadinstitute/hellbender/engine/spark/SAMRecordToGATKReadAdapterSerializerUnitTest.java

```diff
@@ -30,12 +31,21 @@ public void test() {
         KryoSerializer kryoSerializer = new KryoSerializer(conf);
```

Comment:
Re-enable this disabled test, modifying it if necessary to take into account the recent changes.